hazelcast/src/main/java/com/hazelcast/spi/impl/operationservice/impl/operations/Backup.java

```diff
@@ -154,12 +158,12 @@ public void run() throws Exception {
 
     @Override
     public void afterRun() throws Exception {
-        if (!valid || !sync || getCallId() == 0 || originalCaller == null) {
+        if (!valid || !sync || backupOp.getCallId() == 0 || originalCaller == null) {
```

Comment:
Backup op can't be null. I'll have another look at the method. Thanks for the review.